host/settings/settings.go

```diff
@@ -175,8 +175,8 @@ var (
 		BaseRPCPrice:      types.Siacoins(1).Div64(1e6), // 1 SC / million RPCs
 		SectorAccessPrice: types.Siacoins(1).Div64(1e6), // 1 SC / million sectors
 
-		CollateralMultiplier: 2.0, // 2x storage price
-		MaxCollateral:        types.Siacoins(1000),
+		CollateralMultiplier:    2.0,  // 2x storage price
+		MaxCollateralMultiplier: 10.0, // 10x collateral
```

Comment:
Doesn't making this a multiplier effectively remove the max? I'm not sure how you'd apply it in the validation code.

e.g. a host offers 1H / byte / month then the max becomes 10H / byte / month. But what does that even mean? 

In my head that would only work if the multiplier was not just a multiplier but the actual storage you expect a renter to be able to store in a contract. Like `10 * 1<<40` and then the max is 10TB of collateral / month.